services/streammanager/kafka/kafkamanager_test.go

```diff
@@ -800,6 +838,17 @@ func TestSendMessage(t *testing.T) {
 	})
 }
 
+func TestPublish(t *testing.T) {
+	t.Run("no topic publish", func(t *testing.T) {
```

Comment:
The main test is `TestPublish`, no need to repeat `publish`:

```suggestion
	t.Run("no topic", func(t *testing.T) {
```

Reply:
This is addressed